packages/malloy-interfaces/thrift/malloy.thrift

```diff
@@ -348,28 +300,29 @@ struct QueryArrow {
 
 struct QueryRefinement {
   1: required QueryDefinition base,
-  2: required ViewDefinition refinement,
+  2: required ViewDefinition refining_view,
 }
 
 union ViewDefinition {
   1: ViewArrow arrow,
   2: Reference view_reference,
   3: ViewRefinement refinement,
-  4: ViewSegment segment,
+  4: ViewBlock block,
 }
 
 struct ViewRefinement {
   1: required ViewDefinition base,
-  2: required ViewDefinition refinement,
+  2: required ViewDefinition refining_view,
 }
 
 struct ViewArrow {
   1: required ViewDefinition source,
   2: required ViewDefinition view,
 }
 
-struct ViewSegment {
-  1: required list<ViewOperation> operations,
+// TODO: I'm not conviced about just calling this "block"
```

Comment:
Is there a better name to describe a "view block" / "view segment". Internally, we call this a "query operation".

Reply:
MToy doesn't love any of these terms, but is okay with "Block". MToy would like us to pick a name and use it everywhere.